contentcuration/contentcuration/tests/test_task_api.py

```diff
@@ -60,11 +61,11 @@ def test_get_task_list(self):
         response = self.get(url)
         self.assertEqual(len(response.data), 1)
 
-        self.assertEqual(response.data[0]["status"], "STARTED")
-        self.assertEqual(response.data[0]["task_type"], "YOUTUBE_IMPORT")
-        self.assertEqual(
-            response.data[0]["metadata"], {"affects": {"channel": self.channel.id}}
-        )
+        data = response.data[0]
+        self.assertEqual(data["status"], "STARTED")
+        self.assertEqual(data["task_type"], "YOUTUBE_IMPORT")
+        self.assertIsNotNone(data["channel"])
+        self.assertEqual(data["channel"].hex, self.channel.id)
```

Comment:
I don't exactly understand why I needed `.hex` for the value coming from the task api response

Reply:
DRF test tools take a shortcut and don't actually serialize and deserialize, instead passing the raw Python objects out as the response data. I really hate it. So because something was parsed as a UUID, it comes out as a UUID here.